vue-template/package.json

```
@@ -8,11 +8,11 @@
     "preview": "vite preview",
     "lint": "eslint --ext .json,.js,.vue src",
     "lint:fix": "eslint --ext .json,.js,.vue --fix src",
-    "clone:git-actions": "degit Byzanteam/jet-scaffolds/git-actions .github/workflows",
-    "clone:deploy": "degit Byzanteam/jet-scaffolds/deploy deploy",
-    "clone:husky": "degit Byzanteam/jet-scaffolds/husky .husky",
-    "clone:config": "pnpm clone:deploy --force && pnpm clone:git-actions --force && pnpm clone:husky --force",
-    "prepare": "pnpm clone:config && husky install",
+    "clone:deploy": "degit git@github.com:Byzanteam/jet-scaffolds/deploy deploy",
```

`clone` 改成 `setup` 如何？
```suggestion
    “setup:deploy": "degit git@github.com:Byzanteam/jet-scaffolds/deploy deploy",
```